src/backend/InvenTree/InvenTree/ready.py

```diff
@@ -44,6 +44,11 @@ def isRunningBackup():
     )
 
 
+def isGeneratingSchema():
+    """Return true if schema generation is being executed."""
+    return 'schema' in sys.argv
```

Comment:
We probably need to extend this check to also assess if we are using the included api-doc browser